tools/CACertificates/openssl_root_ca.cnf

```diff
@@ -12,6 +12,13 @@ new_certs_dir     = $dir/newcerts
 database          = $dir/index.txt
 serial            = $dir/serial
 RANDFILE          = $dir/private/.rand
+# We enable this solely to facilitate non production and/or test automation environments.
+# This flag essentially allows the creation of certificates with the same subject name
+# (or distinguishing name DN). What this essentially boils down to is if a certificate
+# was created with a specific subject name it can be re-created and thus overwrites
+# the previous certificate and key. Without this openssl fails to re-create the certificate.
+# With this change, cert gen becomes more amenable to CRUD operations during test runs.
+unique_subject    = no
```

Comment:
Does this have any impact if some uses these scripts to generate transparent gateway certs?

Reply:
No it does not. Assuming users follow the instructions as is it is a non issue. What makes this better is that assuming the want to regenerate certificates using the exact same name with other certificate parameters tweaked this change will permit such actions. I have personally needed to do this in the past. Without this change, in order to make the certificate regenerate with the same subject name, I would have to go into the index.txt (this is a DB of all the generated certs) and manually erase the entry.